tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -598,17 +545,18 @@ describe(__filename, () => {
 
     // See this other user profile page.
     const params = { userId: user.id };
```

Comment:
I find all of these `id`s and `userId`s and `user.id`s a bit confusing. It feels like it would be a bit simpler to define `const viewedUserId = 2`, then pass that into `createUserAccountResponse`, and just use it everywhere else, instead of `user.id`.

Reply:
okay, I've gone through and defined the user ids explictly, and then dropped the `user` const where it wasn't needed (e.g. for name)